### ISSUE TITLE
Missing + Incorrect Value labels

### ISSUE THREAD

**Issue by buscandoaverroes** [worldbank/gld#18]
Year 2016 has no complete value labels for LFS I2D2 in any of the four rounds, so this must be built manually.

**Comment by buscandoaverroes** [comment]
update: value labels also missing in virtually all rounds and variables of 2017, but exist in 2018-2020.

**Comment by buscandoaverroes** [comment]
Also, it seems at least some of the value labels are wrong -- or at least inconsistent between the survey documentation and the incoming labels raw dta file. Let me explain.

In 2018, the `pufc07_grade` indicates the highest grade completed. The survey file containing the main factor levels can be found [here](https://www.ilo.org/surveyLib/index.php/catalog/3173/related-materials), which appears to be the same survey file used since 2016. This lists elementary grades to begin at the `200` level, and high school at the `300` level, so grade 1 elementary is `210`, grade 2 `220` and so on.

However, the data suggest that this is not the case. In 2018, `pufc07_grade` is integer, labelled with the following (abbreviated) factor labels:
```
110 = Grade 1 Primary School
120 = Grade 2 Primary School
210 = First Year High School
220 = Second Year High School
``` 

It is likely that this is the case for other survey and other variables, although I have not checked this yet.

1. Where is the most authoritative source for value labels: the incoming `.dta` files, the pdf survey, or elsewhere?
2. If not/since probably not the `.dta` file, what should the workflow be for producing the value labels? Note that there are already situations where there are no data labels in the `.dta` file.

**Comment by buscandoaverroes** [comment]
for now, I will proceed writing the code by using the factor levels that come with the `.dta` file (if they do, such as with 2018), but this is only temporary so I can continue working on there rest of the project.

**Comment by buscandoaverroes** [comment]
This is also true with the occupation codes. In 2018, in theory the data are supposed to coded according to the 2012 occupation code schema. But the partially-coded data for 2018 shows that are many levels and labels in 2018 that don't correspond to existing levels/labels in the 2012 schema. For example, `pufc16_pkb = 68` exists in the data for "real estate" and on the PSA [website](http://psada.psa.gov.ph/index.php/catalog/175/datafile/F1), but this does not exist in the associated code schema.

**Comment by buscandoaverroes** [comment]
Sorry, error on my part. Forget the part about occupation codes -- I was looking at the wrong variable, `pufc16_pkb` refers to industry codes not occupation codes.

**Comment by buscandoaverroes** [comment]
`label dir` and `iecodebook` reveal that there are absolutely no data labels for the first round of 2016 and the labels are incomplete for the rest of the rounds. Essentially must build all value labels manually for this year. But this is somewhat difficult because:

- the only documentation I have is for the whole year of 2016, not per round
- the occupation code label schemes change during 2016
- without data labels, I guess I'm just supposed to assume that the raw data values themselves are accurate (ie `1 = 1` )

**Comment by buscandoaverroes** [comment]
resolved in #43. The conceptual solution is to not address this issue through `iecodebook`, but to come up with a recoding solution/mechanism/algorithm that is entirely independent of `iecodebook` and any value labels than can be utilized across years. This is essentially what the documents in `/Support/Guides and Documentation/I2D2` provide -- a manual algorithm of sorts that allows the coder to know when and how to replace variables with mercurial and unreliable value labels over time
